dev-libs/cgreen/cgreen-1.6.3.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 2020-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake flag-o-matic
+
+DESCRIPTION="Unit test and mocking framework for C and C++"
+HOMEPAGE="
+	https://cgreen-devs.github.io/cgreen/
+	https://github.com/cgreen-devs/cgreen
+"
+SRC_URI="https://github.com/cgreen-devs/cgreen/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="ISC"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="libxml2 test" # REVIEW(fkobi): doc flag could be added
+RESTRICT="!test? ( test )"
+
+DEPEND="libxml2? ( dev-libs/libxml2:= )"
+RDEPEND=${DEPEND}
+BDEPEND="
+	${DEPEND}
+	test? (
+		dev-lang/perl
+		sys-apps/diffutils
+		sys-apps/findutils
+		sys-apps/sed
+	)
+"
+
+PATCHES=( "${FILESDIR}"/${PN}-1.6.3_no-fortify-source.patch )
+
+# NOTE(fkobi): Makefile is a wrapper for cmake, ignore it
```

Comment:
I don't see a need to leave this in. You can put a line in src_configure if you want.

Reply:
I'm saying the comment shouldn't be in "global scope". We also don't generally do "TYPE(person):" comments.

Just put it in src_configure if no src_prepare exists.